packages/batch-submitter/hardhat.config.ts

```diff
@@ -19,7 +19,7 @@ const config: HardhatUserConfig = {
     timeout: 50000,
   },
   solidity: {
-    version: '0.7.0',
+    version: '0.8.9',
```

Comment:
Don't change the batch submitter since this is operating with old set of contracts. There is an ongoing project to rewrite that.